components/signature/overview.md

```diff
@@ -0,0 +1,151 @@
+1. Set the `Width` and `Height` parameters to `px` values.
+
+>caption Blazor Signature with basic configuration
+
+````CSHTML
+<TelerikSignature @bind-Value="@SignatureValue" Width="300px" Height="300px">
+</TelerikSignature>
+
+@code {
+    private string SignatureValue { get; set; }
+}
+````
+
+## Appearance
+
+The Signature component provides settings to control its appearance. [Read more about the Blazor Signature appearance settings...]({%slug signature-appearance%})
+
+## Events
+
+The Blazor Signature **blur** and value **change** events to respond to user actions. [Read more about the Blazor Signature events...]({%slug signature-events%})
+
+## Large Content Support
+
+In **Blazor server-side applications**, the Signature uses the **SignalR WebSocket**, which has a default maximum message size of **32 KB**. To work with larger content, increase the max WebSocket message size for the Blazor application:
+
+>caption Program.cs (.NET 6)
+
+<div class="skip-repl"></div>
+
+````CS
+using Microsoft.AspNetCore.SignalR;
+
+var builder = WebApplication.CreateBuilder(args);
+
+//...
+
+// SignalR message size for the Signature
+builder.Services.Configure<HubOptions>(options =>
+{
+    options.MaximumReceiveMessageSize = 1024 * 1024; // 1MB
+});
+````
+
+>caption Startup.cs (.NET 5)
+
+<div class="skip-repl"></div>
+
+````CS
+public void ConfigureServices(IServiceCollection services)
+{
+    //...
+
+    // SignalR message size for the Signature
+    services.Configure<HubOptions>(options =>
+    {
+        options.MaximumReceiveMessageSize = 1024 * 1024; // 1MB
+    });
+}
+````
+
+## Signature Parameters
+
+The following table lists the Signature parameters. Also check the [Signature API Reference](/blazor-ui/api/Telerik.Blazor.Components.Signature).
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type and Default&nbsp;Value | Description |
+| --- | --- | --- |
+| `AriaDescribedBy` | `string` | Maps to the `area-describedby` attribute. Use this parameter to establish a relationship between widgets or groups and the text that describes them. |
+| `AriaLabel` | `string` | Maps to the `aria-label` attribute. Use  this parameter if the text that labels the component is not visible. |
+| `AriaLabelledBy` | `string` | Maps to the `area-labelledby` attribute. Use this parameter to reference another element to define its accessible name. |
+| `Class` | `string` | Renders a custom CSS class to the `<div class="k-signature">` element. Use it to [override theme styles]({%slug themes-override%}). |
+| `DebounceDelay` | `int` <br /> `(150)` | Time in milliseconds between the last typed symbol and the internal `oninput` event firing. |
+| `Enabled` | `bool` <br /> `(true)` | Defines if the component is enabled. |
+| `ExportScale` | `double` | The `Width` and `Height` of the component will be multiplied by the value of the `ExportScale` when converting the signature to an image. |
```

Comment:
ExportScale has no default value, but in this case, the component will use 2, so we may as well mention it.